recipes/airss/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "airss" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}-pkg
+  version: {{ version }}
+
+source:
+  - url: https://www.mtg.msm.cam.ac.uk/files/airss-{{ version }}.tgz 
+    sha256: b54c61b2988b3e6ee6d01e8b73fc60d76f36ad47eeb459d1a0358e486195cc13
+    patches:
+      - patches/makefile.patch
+  - url: https://github.com/spglib/spglib/archive/refs/tags/v1.16.5.zip
+    sha256: fa8f6eb0dc7bb0410f517038e3a165ce74549d19c41a8bdbb34864c7141304a1
+    folder: external/spglib/spglib-1.16.5
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+outputs:
+  - name: {{ name }}
+    script: build_airss.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('fortran') }}
+        - make
+        - patch
+      host:
+        - spglib 1.16.*
+        - libblas
+        - liblapack
+        - perl
+      run:
+        - perl
+        - {{ pin_compatible('spglib', max_pin='x.x') }}
+    test:
+      commands:
+        - airss
+        - airss version
+        - test ! -f ${PREFIX}/bin/symmol
+        - test -d ${PREFIX}/libexec/airss
+        - test ! -f ${PREFIX}/libexec/airss/symmol
+
+  - name: {{ name }}-with-default-names
+    script: build_default.sh
+    requirements:
+      host:
+        - {{ name }} =={{ version }}
+      run:
+        - {{ name }} =={{ version }}
```

Comment:
You want to use `pin_subpackage( name, exact=True) }}` here so that the pinning includes the build number. Otherwise, you could get a previous build.